### PR TITLE
RDM-3799 New contract for case view component.

### PR DESCRIPTION
**Pull request by andrewfolga**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-3799

### Change description ###
New contract for case view component plus example usage in heroku demo app.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: demo/src/app/case-view-consumer.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'case-create-consumer',
+  template: `<div class="container-fluid">
+                <ccd-case-view [case]="caseId"></ccd-case-view>
+             </div>`
+})
+export class CaseViewConsumerComponent implements OnInit {
```
Comment (gbenadikar) [inline demo/src/app/case-view-consumer.component.ts]: should the name be CaseCreateConsumerComponent?

File: demo/src/app/case-view-consumer.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    caseId;
```
Comment (gbenadikar) [inline demo/src/app/case-view-consumer.component.ts]: type for caseId?

File: demo/src/app/case-view-consumer.component.ts (hunk 1 above)
Comment (andrewfolga) [reply demo/src/app/case-view-consumer.component.ts]: I think this name is actually correct as it is consistent with the other consumers like CaseCreateConsumerComponent or CaseProgressConsumerComponent.

File: demo/src/app/case-view-consumer.component.ts (hunk 2 above)
Comment (andrewfolga) [reply demo/src/app/case-view-consumer.component.ts]: Added.

**Pull request merged**